src/signals_notebook/materials/library.py

```diff
@@ -366,15 +366,24 @@ def create_asset_with_batches(
 
         return cast(ResponseData, result.data).body
 
-    def _is_file_ready(self, report_id: str) -> bool:
+    def _is_file_ready(self, report_id: str) -> dict[str, Any]:
         api = SignalsNotebookApi.get_default_api()
         log.debug('Check job status for: %s| %s', self.__class__.__name__, self.eid)
 
         response = api.call(
             method='GET',
             path=(self._get_endpoint(), 'bulkExport', 'reports', report_id),
         )
-        return response.status_code == 200 and response.json()['data']['attributes']['status'] == 'COMPLETED'
+        response_attributes = response.json().get('data').get('attributes')
```

Comment:
Are you sure this keys will be sent to response?

Reply:
mm, yes. This keys u get always in all responses